demos/segmentation_demo/cpp/main.cpp

```diff
@@ -188,6 +194,33 @@ cv::Mat renderSegmentationData(const ImageResult& result, OutputTransform& outpu
     return output;
 }
 
+void print_raw_results(const ImageResult& result, std::vector<std::string> labels)
```

Comment:
I'd suggest to print image output resolution and total number of pixels here (before statistics). That would ease understanding of what statistics base is used - is is network output or input image resolution.

Reply:
I would agree with you, but similar function in `object_detection_demo` doesnt print such info, assuming that it depends on image, not network, resolution.

Reply:
Here you print number of pixels - but it is not clear - 10 pixels - is it many or not ? We need to know resolution to evaluate that. I wouldn't insist on this change as percentage is printed, but I'd suggest to put such info here (and maybe add it to object_detection_demo too)